src/System.Device.Gpio/System/Device/Gpio/Drivers/SysFsDriver.Linux.cs

```diff
@@ -15,13 +16,41 @@ namespace System.Device.Gpio.Drivers
     public class SysFsDriver : UnixDriver
     {
         private const string GpioBasePath = "/sys/class/gpio";
+        private const string GpioChip = "gpiochip";
+        private const string GpioLabel = "/label";
+        private const string GpioContoller = "pinctrl";
+        private const string GpioOffsetBase = "/base";
         private int _pollFileDescriptor = -1;
         private Thread _eventDetectionThread;
         private int _pinsToDetectEventsCount;
         private readonly static CancellationTokenSource s_eventThreadCancellationTokenSource = new CancellationTokenSource();
         private readonly List<int> _exportedPins = new List<int>();
         private readonly Dictionary<int, UnixDriverDevicePin> _devicePins = new Dictionary<int, UnixDriverDevicePin>();
         private readonly int _pollingTimeoutInMilliseconds = Convert.ToInt32(TimeSpan.FromMilliseconds(1).TotalMilliseconds);
+        private static int _pinOffset = ReadOffset();
+
+        private static int ReadOffset()
+        {
+            IEnumerable<string> fileNames = Directory.EnumerateFileSystemEntries(GpioBasePath);
+            foreach (string name in fileNames)
+            {
+                if (name.Contains(GpioChip))
+                {
+                    try
+                    {
+                        if (File.ReadAllText($"{name}{GpioLabel}").StartsWith(GpioContoller, StringComparison.Ordinal))
+                        {
+                            if (int.TryParse(File.ReadAllText($"{name}{GpioOffsetBase}"), out _pinOffset))
```

Comment:
>_pinOffset [](start = 94, length = 10)

please use a local variable here instead of _pinOffset 

                            if (int.TryParse(File.ReadAllText($"{name}{GpioOffsetBase}"), out int pinOffset))
                                return pinOffset;